packages/hooks/src/useIsP2PEnabled.ts

```diff
@@ -25,6 +25,7 @@ const useIsP2PEnabled = () => {
     return {
         ...rest,
         data: is_p2p_enabled,
+        is_p2p_supported_currency,
```

Comment:
need to return if the user is using a p2p supported currency inside cashier page so we can show the edge case for if the user's currency is not supported. Easier to get the value from here rather than create another hook to return it

Reply:
There is `useHasP2PSupportedCurrencies` to check if user has account with p2p supported currency. but, if you need to check the current currency this approach is correct 🙏